control-plane/pkg/reconciler/sink/kafka_sink.go

```diff
@@ -161,12 +185,62 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, ks *eventing.KafkaSink) r
 
 func (r *Reconciler) finalizeKind(ctx context.Context, ks *eventing.KafkaSink) error {
 
-	// logger := log.Logger(ctx, "finalize", ks)
+	logger := log.Logger(ctx, "finalize", ks)
+
+	// Get sinks config map.
+	sinksConfigMap, err := r.GetOrCreateDataPlaneConfigMap(ctx)
```

Comment:
`contractConfigMap`

Reply:
I'm not sure it's better it just vaguer :thinking:

Reply:
It's more to keep it consistent